tests/chains/tendermint/go.mod

```diff
@@ -168,5 +168,6 @@ require (
 replace (
 	// https://github.com/cosmos/cosmos-sdk/blob/v0.47.3/go.mod#L171-L182
 	github.com/99designs/keyring => github.com/cosmos/keyring v1.2.0
+	github.com/datachainlab/ibc-mock-client => github.com/dongrie/ibc-mock-client v0.3.3-0.20231221102347-9000a7df65b4
```

Comment:
@dongrie I've just created the tag v0.3.3 of ibc-mock-client.
Please use ibc-mock-client@v0.3.3